api/client/webclient.go

```diff
@@ -145,8 +145,12 @@ type SSHProxySettings struct {
 
 // DBProxySettings contains database access specific proxy settings.
 type DBProxySettings struct {
+	// PostgresPublicAddr is advertised to Postgres clients.
+	PostgresPublicAddr string `json:"postgres_public_addr,omitempty"`
 	// MySQLListenAddr is MySQL proxy listen address.
 	MySQLListenAddr string `json:"mysql_listen_addr,omitempty"`
+	// MySQLPublicAddr is advertised to MySQL clients.
+	MySQLPublicAddr string `json:"mysql_public_addr,omitempty"`
```

Comment:
should these public adds be lists of strings, to match the config format?

Reply:
I made it consistent with how other `*_public_addr` fields are handled.